test/api.mocha.coffee

```diff
@@ -172,6 +172,87 @@ describe "API", ->
             expect(group.leader).to.be user._id
             done()
 
+      describe "Party", ->
+        it "can be found by querying for party", (done) ->
+          request.get(baseURL + "/groups/").send(
+            type: "party"
+          ).end (res) ->
+            expectCode res, 200
+
+            party = res.body[0]
+            expect(party._id).to.be group._id
+            expect(party.leader).to.be user._id
+            expect(party.name).to.be group.name
+            expect(party.quest).to.be.eql { progress: {} }
+            expect(party.memberCount).to.be group.memberCount
+            done()
+
+        describe "Chat", ->
+          chat = undefined
+          it "Posts a message to party chat", (done) ->
+            request.post(baseURL + "/groups/" + group._id + "/chat").send(
+              message: "Test MSG"
+            ).end (res) ->
+              expectCode res, 200
+              chat = res.body.message
+              expect(chat.id).to.be.ok
+              expect(chat.timestamp).to.be.ok
+              expect(chat.likes).to.be.empty
+              expect(chat.flags).to.be.empty
+              expect(chat.flagCount).to.be 0
+              expect(chat.uuid).to.be.ok
```

Comment:
Now that I look at it here, this and the following couple of lines should probably compare to the user object.